data/raw/keybindings.json

```
@@ -3721,6 +3721,17 @@
       { "input_method": "keyboard_code", "key": "c", "mod": [ "shift" ] }
     ]
   },
+  {
+    "type": "keybinding",
```

```suggestion
    "type": "keybinding"
```
Resetting test suite